core-aam/index.html

```diff
@@ -548,18 +548,15 @@ <h2>Computed Role</h2>
 &lt;a href="#" role="button"&gt; &lt;!-- computedrole returns "button" --&gt;</pre
             >
           </aside>
-          <p>
-            When an element has a role but is not contained in the required context (for example, an orphaned `listitem` without the required accessible parent of role `list`), User Agents MUST ignore
-            the role token, and return the `computedrole` as if the ignored role token had not been included.
+          <p class="note">
+            When an element has a role but is not contained in the required context (for example, an orphaned `listitem` without the required accessible parent of role `list`), this is an authoring
+            error, but the user agent behavior is not specified as a single rule. For most roles, user agents can either recover the error by ignoring the role, or respect the author's intended role
+            in scenarios deemed by the implementation to be harmless. This unspecified permissiveness in how engines treat author role usage errors is occasionally overridden for a specific role in
+            other specs such as [[HTML-AAM]].
```

Comment:
Hmmm I wish there was a way to word that last line that is a little less vague and vaguely ominious. Maybe we should say something a bit more specific, like:
"This unspecified permissiveness in how engines treat author role usage errors is occasionally overridden for specific HTML tags without ARIA role overrides, as defined in [[HTML-AAM]]"...? Because I think only HTML does this anyway?

Reply:
[Thread continues here](https://github.com/w3c/aria/pull/2284#discussion_r1684768065)... (Prettier bot's fault, not mine.)